fineract-core/src/main/java/org/apache/fineract/infrastructure/jobs/service/JobName.java

```diff
@@ -57,6 +57,7 @@ public enum JobName {
     SEND_ASYNCHRONOUS_EVENTS("Send Asynchronous Events"), //
     PURGE_EXTERNAL_EVENTS("Purge External Events"), //
     PURGE_PROCESSED_COMMANDS("Purge Processed Commands"), //
+    ADD_PERIODIC_ACCRUAL_ENTRIES_FOR_SAVINGS_WITH_INCOME_POSTED_AS_TRANSACTIONS("Add Accrual Transactions For Savings"); //
```

Comment:
Are you sure the job name is correct? Usually there are two types of job:
- Create periodic accruals (ADD_PERIODIC_ACCRUAL_ENTRIES_FOR_SAVINGS)
- Create income postings (ADD_PERIODIC_ACCRUAL_ENTRIES_FOR_SAVINGS_WITH_INCOME_POSTED_AS_TRANSACTIONS vs POST_INTEREST_FOR_SAVINGS)

Which one is needed here? The job name and the description is not really matching as far as i can tell.